implementation/src/main/java/io/smallrye/metrics/SharedMetricRegistries.java

```diff
@@ -212,7 +212,7 @@ private static MeterRegistry resolveMeterRegistry() {
             } catch (ClassNotFoundException | SecurityException | IllegalArgumentException | IllegalAccessException
                     | NoSuchMethodException | InstantiationException | InvocationTargetException e) {
                 LOGGER.logp(Level.SEVERE, CLASS_NAME, METHOD_NAME,
-                        "Encountered exception while reflectively loading Micrometer Prometheus classes: {0}", e);
+                        "Encountered exception while reflectively loading Micrometer Prometheus classes: ", e);
```

Comment:
not clear why you don't need the substitution token

Reply:
The last parameter for the method is `Throwable` and is appended  automatically. The `{0}` just appears in the log statement followed by the `Throwable`/`Exception`